src/components/Films/Films.snapshot.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import Films from './Films';
+
+describe('Films Snapshot', () => {

```

Comment:
you could use https://www.npmjs.com/package/enzyme-to-json package with enzyme to create snapshot tests